app/services/decisions/income_decision_tree.rb

```diff
@@ -24,8 +26,8 @@ def destination
     private
 
     def after_employment_status
-      if not_working
-        if ended_employment_within_three_months
+      if not_working?
+        if ended_employment_within_three_months?
```

Comment:
Added the `?` so they are more like regular ruby predicate methods that explicitly return booleans